### DISCUSSION TITLE
Dependency not loading

### DISCUSSION THREAD

**Discussion by dhruviyengar** [CosmosOS/Cosmos#1814] [Q&A]
No matter what I do, my project icon still shows c#, and not cosmos. Please help me. Visual Studio 2019

**Comment by quajak** [comment]
What steps have you already done? Are the nuget packages installed?

**Reply by dhruviyengar** [reply]
UMMMMMMMMMMMMMMMMMM, can you please tell me how to do that?

**Reply by dhruviyengar** [reply]
I used the user kit.exe installer

**Reply by dhruviyengar** [reply]
nvr mind just looked at the packages issue
